### PR TITLE
Sof-6789 - move render functions into components

### PR DESCRIPTION
**Pull request by VsevolodX**
Should be compared against SOF - 6727

### CODE REVIEW

File: src/components/ParametersMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+            id: "sphere-radius",
+            value: viewerSettings.atomRadiiScale,
+            max: 10,
+            min: 0.1,
+            step: 0.1,
+            onChange: handleSphereRadiusChange,
+            className: "inverse stepper sphere-radius",
+        },
+        {
+            label: "A",
```
Comment (timurbazhirov) [inline src/components/ParametersMenu.tsx]: Repetitions along Vector A

File: src/components/InputWithLabel.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import FormControl from "@mui/material/FormControl";
+import FormLabel from "@mui/material/FormLabel";
+import TextField from "@mui/material/TextField";
+import React from "react";
+
+const defaultFormControlStyle = { margin: "15px", size: "small" };
+const defaultFormLabelStyle = { padding: "5px" };
+const defaultTextFieldStyle = {
+    height: "25px",
+    width: "75px",
```
Comment (timurbazhirov) [inline src/components/InputWithLabel.tsx]: Can be done via theme spacing

File: src/components/IconsToolbar.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+import PowerSettingsNew from "@mui/icons-material/PowerSettingsNew";
+import ButtonGroup from "@mui/material/ButtonGroup";
+import { PaperProps } from "@mui/material/Paper";
+import { useTheme } from "@mui/material/styles";
+import React from "react";
+
+import SquareIconButton from "./SquareIconButton";
+
+// @ts-ignore
+const paperProps: PaperProps = { margin: "0 0 0 15px" };
```
Comment (timurbazhirov) [inline src/components/IconsToolbar.tsx]: Should use theme spacing

Comment (VsevolodX) [reply src/components/IconsToolbar.tsx]: Ok.

**Comment by VsevolodX** [comment]
These changes are expected to fail tests. This branch will merge back to 6727, where the issues are addressed.

**Pull request merged**